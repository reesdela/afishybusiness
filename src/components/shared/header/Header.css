.user-item a {
    font-weight: 700;
    color: #4f4f4f;
    text-decoration: none;
    padding-left: 1.5rem;
}

.header {
    width: 100%;
    background: #f0f0f0;
}

.header-nav {
    max-width: 1000px;
    margin: auto;
}

.header-user {
    float: right;
    margin: auto;
    padding-top: 10px;
    padding-bottom: 10px;
}