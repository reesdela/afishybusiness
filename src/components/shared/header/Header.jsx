import './Header.css'

const Header = () => {
    
    return (
        <div className="header">
            <nav className="header-nav">
                <ul className="header-user">
                    <li className="user-item">
                        <span><a href="./subpages/login/login.php">Sign In</a><a href="./subpages/create_account/create_account.php">Register</a></span>
                    </li>
                </ul>
            </nav>
        </div>
    )
}

export default Header;