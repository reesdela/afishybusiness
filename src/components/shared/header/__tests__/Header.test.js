import renderer from 'react-test-renderer'
import Header from '../Header'
import { MemoryRouter } from 'react-router-dom'

describe('Header', () => {
    it('renders correctly', () => {
        const testRenderer = renderer.create(<MemoryRouter><Header /></MemoryRouter>).toJSON()
        expect(testRenderer).toMatchSnapshot()
    })
})