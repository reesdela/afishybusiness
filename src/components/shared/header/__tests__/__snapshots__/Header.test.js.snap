// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<div
  className="header"
>
  <nav
    className="header-nav"
  >
    <ul
      className="header-user"
    >
      <li
        className="user-item"
      >
        <span>
          <a
            href="./subpages/login/login.php"
          >
            Sign In
          </a>
          <a
            href="/register"
            onClick={[Function]}
          >
            Register
          </a>
        </span>
      </li>
    </ul>
  </nav>
</div>
`;
